src/schema/index.js

```diff
@@ -185,6 +185,7 @@ type Mutation {
     senderAddress: String!
     topicAddress: String!
     oracleAddress: String!
+    optionIdx: Int!
```

Comment:
as discussed, lets keep mutation as close to smart contract interface and grab the optionIdx from the last oracle

Reply:
👍